### PR TITLE
docs: add install instructions

### PR DESCRIPTION
**Pull request by apkrelling**

### CODE REVIEW

**Comment by apkrelling** [comment]
@callumrollo let me know if this looks OK. You might want to check the local pip install instructions - I'm not very familiar with that. Thanks!

File: README.md [hunk 1]
```diff
@@ -1,12 +1,68 @@
...
+First, clone the repo:
+```sh 
+git clone https://github.com/callumrollo/erddaplogs.git
+```
+then you can use Conda:
+```sh
+conda env create --file environment.yml # create the environment 
+conda activate erddaplogs # activate the environment
+```
+or Mamba:
```
Comment (callumrollo) [inline README.md:21]: I don't think the mamba instructions are necessary, those using mamba will know to substitute it for conda

**Review by callumrollo — CHANGES_REQUESTED** [review]
Very thorough! I've added a couple of suggestions

File: README.md [hunk 2]
```diff
@@ -1,12 +1,68 @@
...
+venv ~/virtualenvs/erddaplogs # create the environment
+. ~/virtualenvs/erddaplogs/bin/activate # activate the environment
+
+# go to the directory of the repo, as install the dependencies 
+# as user:
+pip install -r requirements.txt # install the dependencies
+# or as a developer, if you plan on contributing to the project:
+pip install -r requirements-dev.txt # install the dependencies
+```
+
```
Comment (callumrollo) [inline README.md:46]: should be a `pip install .` here at the end i think

File: README.md [hunk 3]
```diff
@@ -1,12 +1,68 @@
...
+* #### From pypi, using pip
+
+```sh
+pip install erddaplogs
```
Comment (callumrollo) [inline README.md:50]: move this one up to the top, simplest and cleanest install method

**Comment by apkrelling** [comment]
Incorporated the suggestions:
- removed instructions for mamba
- moved install from pypi to the top
- changed from `pip install -r requirements-dev.txt` to `pip install .`

**Review by callumrollo — APPROVED** [review]

**Pull request merged**